config/environments/production.rb

```diff
@@ -1,5 +1,20 @@
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
+  config.action_mailer.default_url_options = { :host => 'amy-kang-portfolio.herokuapp.com' }
+
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.default :charset => "utf-8"
+  config.action_mailer.smtp_settings = {
+     :port           => 587,
+     :address        => 'smtp.sendgrid.net',
+     :domain         => 'heroku.com',
+     :enable_starttls_auto => true,
+     :user_name      => ENV['SENDGRID_USERNAME'],
+     :password       => ENV['SENDGRID_PASSWORD'],
+     :authentication => :plain
+ }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.